Tests/UtilityBeltNetworkingTests/Tests/NSNumberExtensionTests.swift

```diff
@@ -0,0 +1,18 @@
+// Copyright © 2021 SpotHero, Inc. All rights reserved.
+
+@testable import UtilityBeltNetworking
+import XCTest
+
+class NSNumberExtensionTests: XCTestCase {
+    func testIsBoolean() {
+        // WHEN: I have a boolean NSNumber
+        let booleanNSNumber = NSNumber(false)
+        // THEN: The `isBoolean` property will return true.
+        XCTAssertTrue(booleanNSNumber.isBoolean)
+        
+        // WHEN: I have a boolean NSNumber
```

Comment:
This WHEN statement is the same as the previous, but the expected outcomes are different.